<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>artists_viewer</class>
 <widget class="QMainWindow" name="artists_viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>artists_viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="universalListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>301</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="universalLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Universal Artists</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="EMILabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>EMI Artists:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="emiListWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>311</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="messageLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
